frontend/public/models/index.ts

```diff
@@ -110,6 +110,28 @@ export const AlertmanagerModel: K8sKind = {
   propagationPolicy : 'Foreground',
 };
 
+export const AlertModel: K8sKind = {
```

Comment:
This is not a native Kubernetes model, and therefore should not be included as `type K8sKind`.

Reply:
Done and moved the models to `alert.tsx`. This means we can't use some of the util helpers like `ResourceIcon`, `resourcePath()` and `referenceForModel()`. I think it's better not to use them anyway since they are really just intended for k8s resources.